CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 - Fix bugs in reading catalogs using pyradiosky for `skyh5` files and files with unusual extension names.
 - Corrects the distribution of random points for the random mock catalog.
 - pixel interpolation was defaulting to az_za_simple for all beams, breaking healpix-coord UVBeams.
-
+- `SkyModel.name` is now coerced to an array.
```

Comment:
Please add this at the top of its section so its in reverse chronological order.

Reply:
Ahhh, I'll never remember which way round that goes. Done.